new/PRS13144GabraWald.xml

```diff
@@ -56,6 +56,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                 </person>
                 <listRelation>
                     <relation name="saws:hasOwned" active="PRS13144GabraWald" passive="ESkae011"/>
+                    <relation name="saws:isRelatedTo" active="PRS13144GabraWald" passive="PRS9451Tewoflos"/>
```

Comment:
This saws relation should only be used between two works, not for persons. What is their exact relationship?